buildpipeline/pipeline.json

```diff
@@ -466,9 +466,9 @@
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
             "PB_DockerTag": "rhel7_prereqs_2",
-            "PB_TargetQueue": "Redhat.72.Amd64",
             "PB_BuildArguments": "-buildArch=x64 -Debug -portableLinux",
-            "PB_SyncArguments": "-p -portableLinux -- /p:ArchGroup=x64"
+            "PB_SyncArguments": "-p -portableLinux -- /p:ArchGroup=x64",
+            "PB_CreateHelixArguments": "/p:ArchGroup=x64 /p:ConfigurationGroup=Debug /p:\"EnableCloudTest=true\" /p:\"TestProduct=corefx /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Linux\" /p:\"TargetQueues=Redhat.72.Amd64;Debian.82.Amd64\"",
```

Comment:
it's fine to leave in EnableCloudTest, but with the changes I made you really just need to not set it to false (it's assumed if you're building this project you want to do some cloud testing): https://github.com/dotnet/corefx/blob/master/src/upload-tests.proj#L124